R/internal_run.R

```diff
@@ -42,6 +42,7 @@ internal_run <- function(input, steps, queue, level,
     } else {
       # Execute the scripts
       queue$run(files)
+      cat("\n")
```

Comment:
All good putting this back in for the nicer console outputs. We should wrap it in a zephyr msg, so that it is only done when verbosity_level is "verbose" or "debug".

Reply:
Good idea